reference/security/defend-advanced-settings.md

```diff
@@ -0,0 +1,963 @@
+
+
+`[linux,mac,windows].advanced.alerts.hash.sha1`
+:   Added in 8.16.0.
+
+    *Compute and include SHA-1 hashes in alerts? This will increase CPU usage and alert sizes. If any user exceptionlists, trustlists, or blocklists reference this hash type, Endpoint will ignore this setting and automatically enable this hash type. Default: `false`.*
+
+    {{elastic-endpoint}} doesn't generate SHA-1 hashes in alerts unless alert exceptions, trusted apps, or blocklisting requires them, in which case this setting is ignored. This setting was added in 8.16 to allow users to opt out of SHA-1 hashing; starting with 8.18, users are opted out by default.
+
+
+`windows.advanced.alerts.rollback.self_healing.enabled`
+:   Added in 8.4.0.
+
+    *Self-healing erases attack artifacts when prevention alerts are triggered. Warning: data loss can occur. Default: `false`.*
+
+    When a prevention alert is generated, {{elastic-endpoint}} can [roll back](/solutions/security/configure-elastic-defend/configure-self-healing-rollback-for-windows-endpoints.md) recent filesystem changes likely associated with the attack. Use this setting to enable the self-healing rollback feature.
+    
+    ::::{warning}
+    This feature can cause permanent data loss.
+    ::::
+
+
+`windows.advanced.alerts.rollback.self_healing.registry_enabled`
+:   Added in 8.8.0.
+
+    *Enables self-healing of registry based malware artifacts. Requires `rollback.self_healing.enabled` to also be enabled. Default: `true`.*
+
+    As an extension to the base-level self-healing rollback feature, {{elastic-endpoint}} can roll back recent registry changes when an attack occurs. Use this setting to enable this feature.
+    
+    ::::{warning}
+    This feature can cause permanent data loss.
+    ::::
+
+
+`windows.advanced.alerts.rollback.self_healing.process_enabled`
+:   Added in 8.8.0.
+
+    *Enables automatic removal of malware processes when a related prevention alert fires, including processes which were not directly involved in the alert. Requires `rollback.self_healing.enabled` to also be enabled. Default: `true`.*
+
+    As an extension to the base-level self-healing rollback feature, {{elastic-endpoint}} can terminate recently spawned processes when an attack occurs. Use this setting to enable this feature.
+
+    ::::{warning}
+    This feature can cause permanent data loss.
+    ::::
+
+
+`[linux,mac,windows].advanced.alerts.sample_collection`
+:   Added in 8.13.0.
+
+    *A value of `false` disables malicious sample collection for alerts. Default: `true`.*
+
+    To help improve future malware detection, Elastic collects samples of unknown malware files for {{ecloud}} users by default. Use this setting to disable the sample collection.
```

Comment:
`To improve the efficacy of malware and reputation protections, Elastic collects samples of unknown malware files. Use this setting to disable the sample collection.`